doc/rvv-intrinsic-spec.adoc

```diff
@@ -651,6 +665,14 @@ The intrinsics provide variants with operand mnemonics of `vv` and `vx`, but not
 
 It is an optimization issue for the implementation to emit instructions with operand mnemonics of `vi` when an immediate that can be expressed within 5-bit is provided to the intrinsics.
 
+=== Mixing inline assembly and intrinsics
+
+The compiler will be conservative to registers (`vtype`, `vxrm`, `frm`) when encountering inline assembly. Users should be aware that mixing uses of intrinsics and inline assembly will result in extra save and restore.
+
+=== The `new_vl` argument in fault-only-first load intrinsics
+
+The fault-only-first load intrinsics write the new value inside the `vl` register into the address of the `new_vl` argument. Providing an illegal memory location will result in a memory access violation.
```

Comment:
Probably should say that an illegal memory location will result in undefined behavior? Otherwise this differs from the way regular C/C++ stores behave. Both clang and gcc already treat a `new_vl == NULL` as undefined behavior and don't necessarily generate a memory access violation.